File: src/main/java/ru/mondayish/kyu8/SimpleCipher.java
package ru.mondayish.kyu8;

public class SimpleCipher {

    public static String decipher(String message) {
        return message.replaceAll("0", "o").replaceAll("1", "i")
                .replaceAll("3", "e").replaceAll("\\$", "s")
                .replaceAll("6", "b").replaceAll("7", "t")
                .replaceAll("-/_", "z").replaceAll("-/", "y")
                .replaceAll("\\)\\(", "x").replaceAll("@", "a")
                .replaceAll("\\|<", "k").replaceAll("\\|\\)", "d")
                .replaceAll("\\|=", "f").replaceAll("\\|-\\|", "h")
                .replaceAll("_!", "j").replaceAll("\\|\\*", "p")
                .replaceAll("\\|", "l");
    }
}

File: src/main/java/ru/mondayish/kyu8/kotlin/SimpleCipher.kt
package ru.mondayish.kyu8.kotlin

object SimpleCipher {

    fun decipher(message: String): String = message.replace("0", "o")
        .replace("1", "i").replace("3", "e").replace("$", "s")
        .replace("6", "b").replace("7", "t").replace("_!", "j")
        .replace("-/_", "z").replace("-/", "y")
        .replace(")(", "x").replace("@", "a")
        .replace("|<", "k").replace("|)", "d")
        .replace("|=", "f").replace("|-|", "h")
        .replace("|*", "p").replace("|", "l")
}